requirements/requirements.txt

```diff
@@ -1,5 +1,4 @@
-numpy==1.15.0
-scikit-learn[alldeps]==0.19.0
-pytest==3.2.1
-pandas==0.22.0
-mxnet==1.3.0b20180820
\ No newline at end of file
+numpy>=1.13.1
+scikit-learn[alldeps]>=0.18.1
+pandas>=0.22.0
+mxnet>=1.0.0
```

Comment:
Regarding mxnet. It's known that datawig doesn't work (tests crash) with versions of mxnet < 1.3